tests/tree-list.js

```diff
@@ -23,6 +24,7 @@ var TREE_LIST = [
         { value: "mozilla-esr45" },
         { value: "mozilla-esr31" },
         { value: "mozilla-esr17" },
+
```

Comment:
Can you revert the addition of the empty line here?
It's always nice to make the change minimal.